ui/app/src/models/operation.ts

```diff
@@ -1,28 +1,14 @@
-import { ResourceType } from "./resourceType";
 import { User } from "./user";
 
 export interface Operation {
-    id: string,
     resourceId: string,
-
-    isActive: boolean,
-    isEnabled: boolean,
     resourcePath: string,
```

Comment:
perhaps you didn't have latest from the feature branch but this operation model already existed? suggest you grab the existing one?

Reply:
updated 👌